README.md

```diff
@@ -1,81 +1,45 @@
-    
-## Contributing
-:tada: First off, thanks for taking the time to contribute! :tada:
-    
-If you would like to contribute to the Mongock project, visit our [official documentation](https://docs.mongock.io/v5/contributing/index.html)
-    
-    
-<a href="https://github.com/mongock/mongock/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=mongock/mongock" />
-</a>
-
-    
-
-[codeOfConduct]: ./CODE_OF_CONDUCT.md
-[mongockLicense]: ./LICENSE.md
-
-[apacheLicense]: http://www.apache.org/licenses/LICENSE-2.0.html
-[documentation_link]: https://docs.mongock.io/
-[github_releases]: https://github.com/mongock/mongock/releases
+# ⚠️ Mongock is Deprecated
+
+**Status:** This project is no longer maintained (as of 2025-08-01).  
+No new features will be added. Critical security fixes may be considered on a best-effort basis only.
+
+> For future-proof projects, we recommend transitioning to [Flamingock](https://github.com/flamingock/flamingock-project).
+
+---
+
+## Meet Flamingock 🚀
+
+We are the original authors of Mongock. Over the past couple of years, we've been heads‑down building **[Flamingock](https://github.com/flamingock/flamingock-project)** — a platform inspired by what worked in Mongock, but with a much broader scope: **auditable, deterministic evolution across your entire system**. (A managed cloud offering is also in the works.)
+
+- Robust support for MongoDB and NoSQL migrations, covering everything previously offered by Mongock
+- Extends beyond databases to orchestrate changes across message queues, contracts, APIs, configs, feature flags, and more
+- Designed **from the ground up with auditability, determinism, and governance at its core**
+
+### More highlights
+- Flexible/low‑code templates for defining changes
+- Native GraalVM support
+- Advanced multi-stage workflows
+- …and more in the docs → [Features overview](https://docs.flamingock.io/docs/overview/key-features)
+
+> Same creators. Broader scope. Better foundations.
+
+👉 **Start here:** [Flamingock repository](https://github.com/flamingock/flamingock-project)
+
+---
+
+## Why the change?
+
+Mongock helped teams evolve NoSQL databases with confidence.  
+But in real-world systems, evolution doesn't stop at databases — it extends to message schemas, APIs, configurations, feature flags, and compliance rules.
+
+**Flamingock was built to support that full spectrum from day one.**  
+It’s not a direct continuation of Mongock, but a new platform that also does everything Mongock did — and much more.
+
+---
+
+## What about existing Mongock users?
+
+- Your current versions will keep working as-is.
+- We strongly recommend evaluating Flamingock for new projects or when planning significant changes.
+- Flamingock introduces a different architecture, but we’ve made it easy to transition when you're ready.
+- Upgrade help: see **[How to move from Mongock to Flamingock](https://docs.flamingock.io/docs/resources/upgrade-from-mongock)**
```

Comment:
Follow our step by step guideline to help you upgrade: see [How to move from Mongock to Flamingock](https://docs.flamingock.io/docs/1.0.0/resources/upgrade-from-mongock).